pkg/util/net/ip/ip.go

```diff
@@ -82,3 +85,28 @@ func isIPv6Disabled(filename string) bool {
 
 	return data[0] == '1'
 }
+
+func ChangeCIDRsToIPs(ifs *api.InterfaceStatus) error {
+	// if IP address was the passed value we should do nothing
+	if net.ParseIP(ifs.Ip) == nil {
+		ip, _, err := net.ParseCIDR(ifs.Ip)
```

Comment:
I would still prefer, like in my initial comment (and david hints it), that if `net.ParseCIRD` fails (it will fail on pure IP addresses without cidr, we should also try to do `net.Parse`.

Reply:
ok, I will do it as you ask. Soon I will update the PR, thanks for review.

Reply:
but just to highlight one thing, I do check `net.ParseIP` before `net.ParseCIDR`, if CIDR (or anything that is not an IP address) is passed to `net.ParseIP` the function will return `nil` (check here https://github.com/kubevirt/kubevirt/pull/4217/files/05eeda4aafcf6c9bc0788da1057a9d69389d8324#diff-4713d8334b0e7a78d689e3a725ab753fR91), and then I check if this is CIDR. So it's the same as you say but in a different order. However I will do it as you wish.

Reply:
If you do that, fine, then I just misseed it.